snippets/omsicfg.json

```diff
@@ -295,14 +346,34 @@
             "$0"
         ]
     },
-    "Transmap": {
-        "prefix": ["matl_transmap", "[matl_transmap]"],
+    "Free Texture": {
+        "prefix": ["matl_freetex", "[matl_freetex]"],
         "body": [
-            "[matl_transmap]",
-            "${1:transmap.bmp}",
+            "[matl_freetex]",
+            "${1:texture}",
+            "${2:variable}",
+            "$0"
+        ],
+        "description": "Marks texture as \"free to change\" and assigns string variable with replacement texture name"
+    },
+    "Texture address": {
+        "prefix": ["matl_texadress", "[matl_texadress]"],
+        "body": [
+            "[matl_texadress_${1|border,clamp,mirror,mirroronce|}]",
```

Comment:
Using the choice system here is a nice idea, but I'm not sure it works given that `border` has 4 parameters (RGBA border colour), while the other three have no parameters.

Reply:
Yes, border mode can be moved to a separate snippet, or just add these 4 color values, because in other modes they will be ignored